observability/grafana/values.yaml

```diff
@@ -0,0 +1,9 @@
+---
+plugins: 'grafana-piechart-panel,grafana-clock-panel,grafana-kubernetes-app'
+
+config:
+  useGrafanaIniFile: true
+  grafanaIniConfigMap: grafana-ini
```

Comment:
I guess in here we could add a list of pre-defined dashboard so we could get better insides of how it works in the future to add more dashboards